src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -242,6 +277,8 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
 
         [OuterLoop] // TODO: Issue #11345
         [ConditionalFact(nameof(BackendDoesNotSupportCustomCertificateHandling))]
+        // For macOS the "custom handling" means that revocation can't be *disabled*. So this test does not apply.
+        [PlatformSpecific(TestPlatforms.Any & ~TestPlatforms.OSX)]
```

Comment:
Yep.  I think I did it to make the `~` more obvious, but I'll go ahead and fold it.